code/modules/awaymissions/corpse.dm

```diff
@@ -21,6 +21,10 @@
 	density = 1
 	anchored = 1
 
+/obj/effect/mob_spawn/New()
```

Comment:
Why didnt you put this with the rest of the New() stuff like 3 lines down.

Or in the spawn/equip proc